spec/features/gardens/index_spec.rb

```diff
@@ -1,20 +1,56 @@
 require 'rails_helper'
 
 RSpec.describe 'Index Page' do
+  describe 'a visitor' do
+    before :each do
+      @garden = { id: 3,
+                attributes: {
+                    name: "Cole Community Garden",
+                    latitude: 39.45,
+                    longitude: -104.58,
+                    description: "A diverse, dedicated group of students and neighbors who believe in bettering ourselves, our food supply and our community through urban gardening.",
+                    private: false },
+                relationships: { plants: {
+                                    data: []},
+                                  users: {
+                                    data: [{id: "4", type: "user"}]},
+                                 sensors: {
+                                    data: []}}}
+    end
+
+    # it 'can visit public gardens page' do
+    #   visit gardens_path
+    #   expect(page).to have_content("Public Gardens")
+    # end
+  end
```

Comment:
Is there a missing test here? Or is that whole describe block not needed?
```suggestion
```

Reply:
Actually do we even have a garden index?

Reply:
We don't! I was being hopeful haha.  I've taken it out!